docs/execution_patterns/EXECUTION.md

```diff
@@ -0,0 +1,41 @@
+# Execution
+
+The Ansible plays in the `splunk-ansible` project are intended to be run locally on each instance/host of the Splunk Enterprise deployment. This means that each host asyncronously sets itself up using Ansible roles to form the final desired topology which is most clearly displayed through the [docker-splunk](https://github.com/splunk/docker-splunk) project. 
```

Comment:
I would probably rewrite this a little.  If the user just reads the first sentence, it makes it seem like the only route would be to run these would be separately on each host.  I suggest something like this.

The Ansible plays in the `splunk-ansible` project can be run in two ways; separately on each instance/host of the Splunk Enterprise deployment or through more traditional separation of control nodes and managed nodes.  In the first method each host asynchronously sets itself up using Ansible roles to form the final desired topology which is most clearly displayed through the [docker-splunk](https://github.com/splunk/docker-splunk) project.  The more traditional methods are outlined below.

Also, `asyncronously` is misspelled in your original doc